main/SQLHelper.cpp

```diff
@@ -7823,28 +7838,10 @@ void CSQLHelper::DeleteDataPoint(const char* ID, const std::string& Date)
 		tLastUpdate.tm_min += 2;
```

Comment:
I think I added this for a specific reason... probably because the date/time also includes seconds

Reply:
I understand.  That's why I did not take it out deletedatapoint. 
For deletedaterange it is not needed.

I added the requirement to have admin rights.

Reply:
OK took them out

Reply:
Is deletedaterange  not the replacement as well for deletedatapoint?

Reply:
Not really a replacement. 
deletedatapoint is used as pre-processor for deletedaterange when a single datapoint must be removed (by keeping the code that adds the extra seconds to the datapoint time)  and to maintain backward compatibility. 
The SQL functionality is moved to deletedaterange